package.json

```
@@ -5,9 +5,8 @@
   "main": "src/index.js",
   "bin": "src/bin.js",
   "scripts": {
-    "format": "prettier --write 'src/**/*.{js,md}' *.{js,md}",
-    "format-verify": "prettier --check 'src/**/*.{js,md}' *.{js,md}",
-    "lint": "eslint src",
+    "lint-fix": "xo --fix && prettier --write '**/*.{yml,md,json}'",
+    "lint": "xo && prettier --check '**/*.{yml,md,json}'",
```

Do we still need to explicitly run Prettier if xo is configured with `prettier: true`? What does `prettier: true` do, then?

```suggestion
    "lint-fix": "xo --fix",
    "lint": "xo",
```

---

XO only runs prettier on JS files, slightly customized to fit its style. This runs Prettier on the rest of them (if you want that)

---

I do want to run Prettier everywhere. Shame xo doesn’t do that itself, the xo && prettier with prettier: true seems real confusing. Could we then use Prettier for all files and not use xo for prettier? I would also like to see prettier as a direct dependency if we use it directly.

---

I could do that, but then I'd have to customize XO to match Prettier’s style, otherwise XO fixes, then Prettier fixes, and XO fails.

However I can definitely add `prettier` as a direct dependency